doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Timers in Node.js
+layout: docs.hbs
+---
+
+# Timers in Node.js and beyond
+
+The Timers module in Node.js contains functions that execute code after a set
+period of time. You do not need to import the Timers module via `require()` as
+all the methods are available globally to emulate the browser JavaScript API.
+To fully understand when timer functions will be executed, it's a good idea to
+read up on the the Node.js
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
+
+## Controlling the Time Continuum with Node.js
+
+The Node.js API provides several ways to schedule your code to execute at
```

Comment:
@jasnell Gotcha. I rewrote all usages of you/your (there were quite a few). Thanks for the feedback!